content/docs/command-reference/repro.md

```diff
@@ -10,14 +10,17 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
-                 [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull] [--glob]
+                 [targets [<target> ...]]
```

Comment:
I don't see this `<targets>` in the help output on the latest master branch @skshetry was that something you added or is it how it will look in 2.0? Thanks
```suggestion
                 [targets [targets ...]]
```

Reply:
`target` is a spec, so either you emphasize it with `<target>` as:
```
                 [targets [<target> ...]]
```

or, just with the `target`. `targets [targets ...]` don't make sense.

> I don't see this <targets> in the help output on the latest master branch

I don't have any problem with changing this on the DVC side. The only reason I made `<target>` here is to emphasize it's more of a pattern/syntax. Also, note that this kind of syntax (except glob) is supported by `push/pull/checkout` commands, but as we are moving away from using stage name in those commands, I only added them in the `repro` where it makes the most sense.

> is it how it will look in 2.0?

This has been the case since 1.0 (except glob of course).